### PR TITLE
[RDY] Added documentation for two more functions

### PR DESCRIPTION
**Pull request by PedroBortolli**
Added missing documentation for 2 functions of `api.txt`:
* nvim_get_color_by_name
* nvim_get_color_map

### CODE REVIEW

**Comment by mhinz** [comment]
`api.txt` is generated by [gen_vimdoc.py](https://github.com/neovim/neovim/blob/master/scripts/gen_vimdoc.py). The descriptions need to be added to [api/vim.c](https://github.com/neovim/neovim/blob/master/src/nvim/api/vim.c#L1201).

**Comment by PedroBortolli** [comment]
> `api.txt` is generated by [gen_vimdoc.py](https://github.com/neovim/neovim/blob/master/scripts/gen_vimdoc.py). The descriptions need to be added to [api/vim.c](https://github.com/neovim/neovim/blob/master/src/nvim/api/vim.c#L1201).

Oh, my bad. I will add them to the correct file and bump this PR once it's done.

**Comment by PedroBortolli** [comment]
Okay, re-opening the PR as now the documentation is where it's supposed to be.

File: src/nvim/api/vim.c [hunk 1]
```diff
@@ -1198,12 +1198,25 @@ void nvim_unsubscribe(uint64_t channel_id, String event)
...
+///         find a correct RGB value for the given color name.
 Integer nvim_get_color_by_name(String name)
   FUNC_API_SINCE(1)
 {
   return name_to_color((char_u *)name.data);
 }
 
+/// Returns a map (dictionary) with all colors from rgb.txt.
+/// Every key in the map is a color name (e.g. “Aqua”), and every
+/// value is an RGB decimal value (e.g. 65535).
```
Comment (bfredl) [inline src/nvim/api/vim.c]: 24-bit RGB value

**Pull request merged**